src/config/config.cc

```diff
@@ -110,6 +110,7 @@ Config::Config() {
       {"daemonize", true, new YesNoField(&daemonize, false)},
       {"bind", true, new StringField(&binds_str_, "")},
       {"port", true, new UInt32Field(&port, kDefaultPort, 1, PORT_LIMIT)},
+      {"socket-fd", true, new IntField(&socket_fd, -1, -1, 1 << 16)},
```

Comment:
We need to add `socket-fd` into `kvrocks.conf` and add some comments to describe it.

Also we need to note to users that: if it's specified, `bind` and `tls*` cannot work.